### DISCUSSION TITLE
nu_plugin_chart

### DISCUSSION THREAD

**Discussion by mvolkmann** [nushell/nushell#3161] [Q&A]
Where can I find documentation on using nu_plugin_chart or examples of using it?

**Comment by andrasio** [comment]
@mvolkmann Sorry about that. Will get on that. This could help though, a bit of [examples](https://github.com/nushell/nushell/pull/2627) in [pull requests](https://github.com/nushell/nushell/pull/2621).

**Comment by mvolkmann** [comment]
I have this simple CSV file in `scores.csv`:

```css
Name,Score
Mark,19
Tami,21
Amanda,17
Jeremy,15
```

How can I display this as a bar chart where the names are on the x-axis and the scores are the heights of the bars?
I tried this, but it renders all the bars with a value of either 100 or 0.

```bash
open scores.csv | chart bar [Score, Name]
```
